views/styles/chart_styles.js

```diff
@@ -146,12 +146,16 @@ export const chartStyles = (theme) => ({
   },
   viewChartRow: {
     display: 'flex',
-    justifyContent: 'space-between',
+    flexDirection: 'column',
```

Comment:
```suggestion
    flexDirection: 'column',
    gap: 2px 0;
```

100% optional! You could set a [gap](https://developer.mozilla.org/en-US/docs/Web/CSS/gap) here, and then ditch the `paddingTop: 2px` below.